packages/eslint-plugin/src/rules/pkg-renaming.js

```diff
@@ -30,6 +30,39 @@ export default {
             fix: fixer => fixer.replaceText(node.source, `'${maybeMapping}'`)
           })
         }
+        if (
+          (node.source.value === '@emotion/css' ||
+            node.source.value === '@emotion/css/macro') &&
+          node.specifiers.length === 1 &&
+          node.specifiers[0].type === 'ImportDefaultSpecifier'
+        ) {
+          let replacement =
+            node.source.value === '@emotion/css'
+              ? '@emotion/react'
+              : '@emotion/react/macro'
+          context.report({
+            node: node.source,
+            message: `The default export of "${
+              node.source.value
+            }" in Emotion 10 has been moved to a named export, \`css\`, from "${replacement}" in Emotion 11, please import it from "${replacement}"`,
+            fix: fixer =>
+              fixer.replaceText(
+                node,
+                `import { css${
+                  node.specifiers[0].local.name === 'css'
+                    ? ''
+                    : ` as ${node.specifiers[0].local.name}`
+                } } from '${replacement}'`
+              )
+          })
+        }
+        if (node.source.value === 'emotion-theming') {
+          context.report({
```

Comment:
should we bother with "merging" with existing import declarations? Or do you think this is out of the scope of this plugin?

Reply:
I think it's out of scope for now because I feel like that would cause problems because of the way ESLint fixers are run in editors.

Reply:
I think it's out of scope for now because I feel like that would cause problems because of the way ESLint fixers are run in editors.